docs/BuildFromSource.md

```diff
@@ -224,13 +266,15 @@ On macOS/Linux:
 ./build.sh --test
 ```
 
-## Building a subset of the code
+> :bulb: If you're working on changes for a particular subset of the project, you might not want to execute the entire test suite. Instead, only run the tests within the subdirectory where changes were made.
```

Comment:
Does `build.cmd` have a way to execute a particular subset of test? or do we need to activate and then do `dotnet test`?

Reply:
Yes, e.g. `.\build.cmd -test -projects .\src\Framework\test\Microsoft.AspNetCore.App.UnitTests.csproj`. You could add `-noBuild` to that command line if the build is already completely up-to-date.

Reply:
Awesome! thanks.